examples/va/nfv/sriov/edpm/values.yaml

```diff
@@ -94,18 +34,15 @@ spec:
         service_net_map:
           nova_api_network: internal_api
           nova_libvirt_network: internal_api
-        # CHANGEME
         timesync_ntp_servers:
           - hostname: clock.redhat.com
         # CPU pinning settings
-        edpm_kernel_args: "default_hugepagesz=1GB hugepagesz=1G hugepages=16 intel_iommu=on iommu=pt isolcpus=4-23,28-47" # CHANGEME
+        edpm_kernel_args: "default_hugepagesz=1GB hugepagesz=1G hugepages=16 intel_iommu=on iommu=pt isolcpus=4-23,28-47"
         edpm_tuned_profile: "cpu-partitioning" 
-        edpm_tuned_isolated_cores: "4-23,28-47" # CHANGEME
+        edpm_tuned_isolated_cores: "4-23,28-47"
```

Comment:
edpm_tuned_isolated_cores in general should include the cores in both
cpu_shared_set and cpu_dedicated_set as well as any core usef for example ovs-dpdk

in this sriov va i think the best approach would be to either not set
edpm_tuned_isolated_cores or set it to 

```
edpm_tuned_isolated_cores: "3-48"
``

Reply:
Okay, I will use the range you suggest as opposed to leaving it out.  That way we are demonstrating the use of the variable.